openhands/core/config/sandbox_config.py

```diff
@@ -31,6 +32,7 @@ class SandboxConfig:
     """
 
     api_hostname: str = 'localhost'
+    remote_runtime_url: str = 'http://localhost:8000'
```

Comment:
in the RemoteRuntime class is a "port" (60000), is that a different one?

Reply:
Yup--this is the port that the EventStream API server runs on, inside the runtime container